src/datahike/pull_api.cljc

```diff
@@ -1,12 +1,13 @@
 (ns ^:no-doc datahike.pull-api
   (:require
-    [datahike.db :as db]
-    [datahike.datom :as dd]
-    [datahike.pull-parser :as dpp #?@(:cljs [:refer [PullSpec]])])
-    #?(:clj
-      (:import
-        [datahike.datom Datom]
-        [datahike.pull_parser PullSpec])))
+   [datahike.db :as db]
+   [datahike.datom :as dd]
+   #?@(:cljs [datalog.parse.type :refer [PullSpec]])
+   [datalog.parser.pull :as dpp])
```

Comment:
The alias `dpp` is never used later in the file.

Reply:
I adjusted that.